lua/autorun/server/bounties.lua

```diff
@@ -1,8 +1,12 @@
 
+BountyLimit = {}
+
 hook.Add("PlayerDeath","BountyDeath",function(ply,infl,atk)
 	local bounty = GetPlayerBounty(ply)
+	if BountyLimit[ply:SteamID()] == nil then BountyLimit[ply:SteamID()] = 0 end
 	if bounty > 0 and ply != atk and atk:IsPlayer() then
 		SetPlayerBounty(ply,0)
+		BountyLimit[ply:SteamID()] = math.Clamp(BountyLimit[ply:SteamID()] - bounty,0,10000000)
```

Comment:
can use math.max instead of math.clamp because this only lowers the bounty limit